### PR TITLE
Change requests from prev PR are resolved here

### LINKED ISSUE #50: Linked event don't open in a new page on item click
Linked events should be opened in a new page once one of them is clicked.

### PR DESCRIPTION
**Pull request by Daniilmass**
resolves #50

### CODE REVIEW

**Comment by Daniilmass** [comment]
Had to recreate PR #51  to avoid multiple conflicts with updates since that

File: src/components/pages/SingleEventPage.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  /**
+   * Renders single LinkedEvent content content
+   */
+  private renderEventContent() {
+    const { classes, locationKey } = this.props;
+    const { fetchedContent } = this.state;
+    if (!fetchedContent) {
+        return null;
+    } else {
+        return(
```
Comment (Kaemppi) [inline src/components/pages/SingleEventPage.tsx:79]: check the indentation, one tab should be 2 spaces

File: src/components/pages/SingleEventPage.tsx [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    }
+  }
+
+  /**
+   * Loads a single LinkedEvent content
+   */
+  private getEventWithId = async () => {
+    const { eventId } = this.props;
+    if (!eventId || eventId == null) {
+        console.log("Event id is: ", eventId);
```
Comment (Kaemppi) [inline src/components/pages/SingleEventPage.tsx:121]: check the indentation

File: src/styles/single-event-page.ts [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    * Hero-image as background 
+    */
+  heroImageDiv: {
+    backgroundImage: `url(${hero})`,
+    backgroundRepeat: "no-repeat",
+    backgroundPosition: "center center",
+    backgroundSize: "cover",
+    color: "#fff",
+    height:"32vh",
+    maxHeight:"30vw",
```
Comment (Kaemppi) [inline src/styles/single-event-page.ts:41]: don't use vh (viewport width) as a unit in heights. Instead use vh or percentage or fixed pixels

**Pull request merged**